python/paddle/fluid/framework.py

```diff
@@ -1137,6 +1137,8 @@ def __init__(self, block, shape, dtype, **kwargs):
 
         self.gradient_clip_attr = kwargs.get('gradient_clip_attr', None)
 
+        self.average = kwargs.get('average', True)
```

Comment:
average -> do_model_ average ?

The meaning of average is not clear.

Reply:
Done.